src/components/global/Navbar.jsx

```diff
@@ -156,7 +162,11 @@ function NavBar() {
           />
         </Link>
         <Typography component="h1" variant="h4" display={{ xs: 'none', sm: 'block' }} pl={'10px'}>
-          <Link to="/" style={navTextStyle} aria-label="CODE PDX">
+          <Link
+            to="/"
+            style={{ ...navTextStyle, color: theme.palette.primary.contrastText }}
+            aria-label="CODE PDX"
+          >
             CODE PDX
           </Link>
         </Typography>
```

Comment:
The 'CODE PDX' (and other nav elements in header) are now black on light mode but should remain white as in current website.

Reply:
Ok with this change as white failed contrast standards so this is better for now.